stl/inc/random

```diff
@@ -508,6 +508,34 @@ _NODISCARD _Uint _Next_linear_congruential_value(_Uint _Prev) noexcept {
     }
 }
 
+// Optimized specialization for minstd_rand0
+template <>
+_NODISCARD inline unsigned int _Next_linear_congruential_value<unsigned int, 16807, 0, 2147483647>(

```

Comment:
It seems clearer to merge these 2 cases into the primary template with `if constexpr`. (We're widely using `if constexpr` in C++14 mode.)